code-challenges.test.js

```diff
@@ -15,42 +15,131 @@
+        expect(orderedOddArray(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(orderedOddArray(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
 
+// ReferenceError: orderedOddArray is not defined
 
 // b) Create the function that makes the test pass.
 
+// Pseudo ---->
+// Create a function named orderedOddArray
+// Pass an array paramater
+// filter over the array using the filter() method 
+// Compare the data type of the value strictly equal to a number data type using typeof 
+// use and operator to set another condition for the new array
+// use modulo to determine if the value is odd
+// If it is odd then use the sort method to properly set its index in the new array
+// Set the sort method to <(a,b)=>a-b> if a results in negative one it is smaller if b is negative one it is smaller.
+// Outside of testing console log the function and paramater to see the result in the terminal
+
+ const orderedOddArray = (array) => array.filter(value => typeof value === "number" && value % 2 !== 0).sort((a,b)=>a-b)
+
+
+// PASS  ./code-challenges.test.js
+// orderedOddArray
+// ✓ passes in an array and returns an array with odd numbers from least to greatest
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersToAdd1 = [2, 4, 45, 9]
-// Excpected output: [2, 6, 51, 60]
+// input: numbersToAdd1
+// output: [2, 6, 51, 60]
+
+// input: numbersToAdd2
+// output: [0, 7, -1, 11]
 
-const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
+// input: numbersToAdd3
+// []
 
-const numbersToAdd3 = []
-// Expected output: []
+describe("accumulatingSum", () => {
+    it(" returns an array of the accumulating sum of array being passed to the function", ()=> {
 
+        const numbersToAdd1 = [2, 4, 45, 9]
+        // Excpected output: [2, 6, 51, 60]
+
+        const numbersToAdd2 = [0, 7, -8, 12]
+        // Expected output: [0, 7, -1, 11]
+
+        const numbersToAdd3 = []
+        // Expected output: []
+
+        expect(accumulatingSum(numbersToAdd1)).toEqual([2, 6, 51, 60])
+        //expect(accumulatingSum(numbersToAdd2)).toEqual([0, 7, -1, 11])
+        expect(accumulatingSum(numbersToAdd3)).toEqual([])
+    })
+})
+
+
+// ReferenceError: sumOf is not defined
 
 // b) Create the function that makes the test pass.
+
+// Pseudo ---->
+// Create a function named accumulatingSum
+// Pass in an array paramater
+// map over the array to access the values
+// add the indexes value to a newVal variable and store it back in the newVal variable
+// set the default newVal variables value to zero at the start when the function is invoked
+// Outside of testing console log the function and paramater to see results in the terminal
+
+const accumulatingSum = (array) => array.map(value =>  newVal += value ,newVal = 0)
```

Comment:
You have some really cool approaches to these problems, I would really love to see you volunteer some of these approaches more often when we go through this as a class.